scripts/sanitycheck

```diff
@@ -605,17 +607,22 @@ class BinaryHandler(Handler):
 
         subprocess.call(["stty", "sane"])
         self.instance.results = harness.tests
-        if not self.terminated and self.returncode != 0:
+
+        if self.returncode != 0:
             #When a process is killed, the default handler returns 128 + SIGTERM
```

Comment:
This comment was meant to explain why the if above had the condition "if not self.terminated"
Now, this comment, even that it is informative, is a bit irrelevant (Meaning, you can delete it if you want)

Anyhow with the 60 seconds timeout is unlikely we will be terminating them for anything more than hung tests.